README.md

```diff
@@ -10,3 +10,30 @@ To add or modify grafana's panels, look in `data/grafana`.
 
 Deployment is done via Github Actions, it's simply a rsync of the repo to
 the home directory of the user on the stats' server.
+
+## Systemctl
+
+All measurements are triggered by systemctl. If you remove one of the targets
+in `.config/systemd/user/timers.targets.wants`, you'll have to remove it by hand:
+
+```bash
+systemctl --user stop `timer_type@name_of_service`
+```
```

Comment:
I don't think so, the deploy workflow should take care of this.

Reply:
I only see a `systemctl --user daemon-reload` in the deploy workflow. Does that stop all timers?

Reply:
```
       daemon-reload
           Reload the systemd manager configuration. This will rerun all generators (see
           systemd.generator(7)), reload all unit files, and recreate the entire
           dependency tree. While the daemon is being reloaded, all sockets systemd
           listens on behalf of user configuration will stay accessible.
```

Reply:
I don't understand if this will stop all timers. Because the deploy script only goes through all timers and starts them. So if `daemon-reload` doesn't stop a removed timer, then the deploy script doesn't take care of this.

Reply:
I thought so but I tried it out and indeed doesn't remove the already running timers. Can you add a deploy step to do so then?

Reply:
I'm merely maintaining, but I let the ownership of the stats package to you ;) If I'd redo it, I'd use cronjobs anyway...

Opened #28